lib/src/base-styles/_nx-colors.scss

```
@@ -170,6 +187,19 @@
     // NxCounter
     --nx-color-counter: var(--nx-swatch-indigo-90);
     --nx-color-counter-active-text: var(--nx-swatch-indigo-05);
+
+    //NxProgressBar
+    --nx-color-progress-bar-text-success: var(--nx-swatch-green-50);
+    --nx-color-progress-bar-icon-success: var(--nx-swatch-green-60);
+    --nx-color-progress-bar-text-error: var(--nx-swatch-red-80);
+    --nx-color-progress-bar-icon-error: var(--nx-swatch-red-60);
+    --nx-color-progress-bar-counter-background: var(--nx-swatch-indigo-80);
+    --nx-color-progress-bar-counter-border: var(--nx-color-progress-bar-counter-background);
+    --nx-color-progress-bar-counter-percent: var(--nx-swatch-indigo-05);
```

```suggestion
    --nx-color-progress-bar-counter-percent: var(--nx-color-counter-active-text);
```
To me, the text color is still intimately tied to NxCounter, and wouldn't hurt to have linked in the variable instead of the more generic `indigo-05`

---

> To me, the text color is still intimately tied to NxCounter, and wouldn't hurt to have linked in the variable instead of the more generic `indigo-05`

I'm leaning towards keeping it at `indigo-05`. I agree that nxProgressBar's counter text color is intimately tied to NxCounter's in light mode. But it seems these counters differ in text color in dark mode.

I see that both `--nx-color-progress-bar-counter-percent` and `--nx-color-counter-active-text` both have the same color swatch. I'm curious if the two variables are connected together, that it would make someone think `--nx-color-progress-bar-counter-percent` is used for an active state, which its not. @rpokorny, can I get your thoughts on this?

---

> I'm leaning towards keeping it at indigo-05. I agree that nxProgressBar's counter text color is intimately tied to NxCounter's in light mode. But it seems these counters differ in text color in dark mode.
> 
> I see that both --nx-color-progress-bar-counter-percent and --nx-color-counter-active-text both have the same color swatch. I'm curious if the two variables are connected together, that it would make someone think --nx-color-progress-bar-counter-percent is used for an active state, which its not. @rpokorny, can I get your thoughts on this?

I agree with @achen5 .  The dark mode styles for the progress counter don't cleanly match those for regular `NxCounter` (have we asked @JasonDSwearingen why not?).  That the text color here is the same as the _active_ text color in `NxCounter` (noting that in light mode, it is the inactive `NxCounter` that matches) seems like a coincidence to me.

---

There were some reasons, but none of them supercede this issue. I have adjusted the mock so that the percentage `Counter` in `NxProgressBar` is the default (rather than active) state.

---

> There were some reasons, but none of them supercede this issue. I have adjusted the mock so that the percentage `Counter` in `NxProgressBar` is the default (rather than active) state.

@rpokorny @EmilyBermudez88,
I updated the counter colors based on the new mockup changes: [b391c63](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/b391c63d4081b1eee83bfd2ee3bb18383c69cc81)

---

> > There were some reasons, but none of them supercede this issue. I have adjusted the mock so that the percentage `Counter` in `NxProgressBar` is the default (rather than active) state.
> 
> @rpokorny @EmilyBermudez88, I updated the counter colors based on the new mockup changes: [b391c63](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/b391c63d4081b1eee83bfd2ee3bb18383c69cc81)

This looks good, although in the mocks the inline counter needs to be changed to reflect the new styling as well.

---

> > > There were some reasons, but none of them supercede this issue. I have adjusted the mock so that the percentage `Counter` in `NxProgressBar` is the default (rather than active) state.
> > 
> > 
> > @rpokorny @EmilyBermudez88, I updated the counter colors based on the new mockup changes: [b391c63](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/b391c63d4081b1eee83bfd2ee3bb18383c69cc81)
> 
> This looks good, although in the mocks the inline counter needs to be changed to reflect the new styling as well.

Fixed: https://www.sketch.com/s/2393264e-1233-4909-99c8-a7dda02588cd/a/3oOadnd